wandb/sdk/data_types/image.py

```diff
@@ -333,6 +333,17 @@ def _initialize_from_data(
         ), f"file_type must be one of {accepted_formats}"
         tmp_path = os.path.join(MEDIA_TMP.name, runid.generate_id() + "." + self.format)
         assert self._image is not None
+
+        # JPEG format does not support transparency.
+        # So we remove the alpha (transparency) channel.
```

Comment:
this comment is unnecessary because it is the same as the warning a couple lines down